addons/l10n_cl/views/report_invoice.xml

```diff
@@ -229,4 +264,98 @@
         </xpath>
     </template>
 
-</odoo>
+    <template id="cl_document_tax_totals" inherit_id="account.document_tax_totals">
+
+        <t t-foreach="tax_totals['subtotals']" t-as="subtotal" position="before">
+            <t t-set="total_amounts" t-value="o._l10n_cl_get_amounts()"/>
+            <t t-set="withholdings" t-value="o._l10n_cl_get_withholdings()"/>
+        </t>
+        <t t-foreach="tax_totals['subtotals']" t-as="subtotal" position="replace">
+            <t t-if="o.country_code != 'CL' or o.l10n_latam_document_type_id._is_doc_type_electronic_ticket()">
+                <t t-foreach="tax_totals['subtotals']" t-as="subtotal">
+                    <tr class="border-black o_subtotal">
+                        <td><strong t-esc="subtotal['name']"/></td>
+                        <td class="text-end">
+                            <span t-att-class="oe_subtotal_footer_separator" t-esc="subtotal['formatted_amount']"/>
+                        </td>
+                    </tr>
+                    <t t-set="subtotal_to_show" t-value="subtotal['name']"/>
+                    <t t-call="account.tax_groups_totals"/>
+                </t>
+            </t>
+            <t t-else="">
+                <tr class="border-black o_subtotal" t-if="total_amounts['subtotal_amount_taxable'] != 0">
+                    <td class="border-black o_subtotal"><strong>Amount Net</strong></td>
+                    <td class="text-end o_price_total">
+                        <span t-att-class="oe_subtotal_footer_separator"
+                              t-esc="total_amounts['subtotal_amount_taxable']"
+                              t-options="{'widget': 'monetary', 'display_currency': total_amounts['main_currency']}"
+                        />
+                    </td>
+                </tr>
+
+                <tr class="border-black o_subtotal" t-if="total_amounts['subtotal_amount_exempt'] != 0">
+                    <td class="border-black o_subtotal"><strong>Amount Exempt</strong></td>
+                    <td class="text-end o_price_total">
+                        <span t-att-class="oe_subtotal_footer_separator"
+                              t-esc="total_amounts['subtotal_amount_exempt']"
+                              t-options="{'widget': 'monetary', 'display_currency': total_amounts['main_currency']}"
+                        />
+                    </td>
+                </tr>
+
+                <tr class="border-black o_subtotal" t-if="total_amounts['vat_amount'] != 0">
+                    <td class="border-black o_subtotal">
+                        <strong t-esc="'VAT %s' % total_amounts['vat_percent']"/><strong>%</strong>
+                        <strong>( Base: </strong>
```

Comment:
With this changes, the footer is being shown like this:
<img width="424" alt="Captura de pantalla 2023-10-02 a la(s) 09 30 13" src="https://github.com/odoo/odoo/assets/382893/ba226dfe-5070-469d-a789-cbeafab58efd">

Nevertheless: the example in the documentation gives this detail. It does not say anything about "base" amounts but it does not forbid it:
<img width="1157" alt="Captura de pantalla 2023-10-02 a la(s) 09 31 05" src="https://github.com/odoo/odoo/assets/382893/e4522804-1a91-45f1-9e3a-0f8a402e4fef">